package.json

```diff
@@ -137,6 +139,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-is": "^17.0.2",
+    "react-refresh": "0.9",
```

Comment:
Isn't that already in place thanks to `react-scripts`?
I know we've probably discussed this many times but won't having it here just cause another problem later when `react-scripts` decides it needs `react-refresh@1.0`?

Reply:
It already complains if it doesn't have the version it likes 😆 , that's why I had to pin the `babel-loader` version that Storybook wanted, but it didn't like